### DISCUSSION TITLE
אולי טעות העתקה בתלכ"צ גליון 2 שאלה 2 סעיף 7

### DISCUSSION THREAD

**Discussion by micha-sherman** [NuclearGandhi/technion_second_brain#10] [טעויות ותיקונים]
אולי גם הפעם אני טועה אבל אני חושב שהייתה לך איפשהו טעות בהצבה
![image](https://github.com/user-attachments/assets/9a8be067-55c3-407f-8e79-d079f7c85449)
![Uploading image.png…]()

**Comment by micha-sherman** [comment]
![WhatsApp Image 2024-08-21 at 11 19 42 AM](https://github.com/user-attachments/assets/55fad176-9b3c-41b6-ac8c-acaf85bc993c)

**Reply by NuclearGandhi** [reply]
צריך להכפיל את מה שקיבלת ב- $`\frac{1}{60\cdot1000}`$, עניין של המרת יחידות. מה שקיבלת שם זה לא ב-Watt, אלא בניוטון מילימטר לדקה. Watt זה ניוטון מטר לשניה.

**Reply by micha-sherman** [reply]
צודק, תודה
